### PR TITLE
Add Hausa language

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review :) please don't forget to add Hausa to Mission Control :)

File: unicorecmsgem/locale/ara_AE/LC_MESSAGES/unicorecmsgem.po [hunk 1]
```diff
@@ -17,6 +17,33 @@ msgstr ""
...
 
+#. Default: We can't find that page, sorry.
+#: unicorecmsgem/templates/404.pt:7
+msgid "We can't find that page, sorry."
+msgstr ""
+
+#. Default: Home Page
+#. Used in sentence: "Go to the <dynamic element>"
+#: unicorecmsgem/templates/404.pt:10
+#, fuzzy
```
Comment (miltontony) [inline unicorecmsgem/locale/ara_AE/LC_MESSAGES/unicorecmsgem.po]: fuzzy

File: unicorecmsgem/locale/ben_BD/LC_MESSAGES/unicorecmsgem.po [hunk 2]
```diff
@@ -17,6 +17,33 @@ msgstr ""
...
 
+#. Default: We can't find that page, sorry.
+#: unicorecmsgem/templates/404.pt:7
+msgid "We can't find that page, sorry."
+msgstr ""
+
+#. Default: Home Page
+#. Used in sentence: "Go to the <dynamic element>"
+#: unicorecmsgem/templates/404.pt:10
+#, fuzzy
```
Comment (miltontony) [inline unicorecmsgem/locale/ben_BD/LC_MESSAGES/unicorecmsgem.po]: fuzzy

File: unicorecmsgem/locale/ben_IN/LC_MESSAGES/unicorecmsgem.po [hunk 3]
```diff
@@ -17,6 +17,33 @@ msgstr ""
...
 
+#. Default: We can't find that page, sorry.
+#: unicorecmsgem/templates/404.pt:7
+msgid "We can't find that page, sorry."
+msgstr ""
+
+#. Default: Home Page
+#. Used in sentence: "Go to the <dynamic element>"
+#: unicorecmsgem/templates/404.pt:10
+#, fuzzy
```
Comment (miltontony) [inline unicorecmsgem/locale/ben_IN/LC_MESSAGES/unicorecmsgem.po]: fuzzy

**Comment by miltontony** [comment]
More fuzzy to remove and double check

**Comment by miltontony** [comment]
I've added Hausa to Mission Control

**Comment by smarzban** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
:+1:

**Pull request merged**